### PR TITLE
Dilpreet | Test |  Design Issue of Version 1

### PR DESCRIPTION
**Pull request by dilpreet77**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
https://app.clickup.com/t/1xbwu8d

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/userDetails/components/move-company/move-company.component.ts [hunk 1]
```diff
@@ -30,10 +35,8 @@ export class MoveCompanyComponent implements OnInit, OnDestroy {
     };
     /** Subject to release subscription memory */
     private destroyed$: ReplaySubject<boolean> = new ReplaySubject(1);
-    /* This will hold local JSON data */
-    @Input() public localeData: any = {};
-    /* This will hold common JSON data */
-    @Input() public commonLocaleData: any = {};
+    /** control for the MatSelect filter keyword */
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/userDetails/components/move-company/move-company.component.ts:38]:
```suggestion
    /** Control for the MatSelect filter keyword */
```

**Review by prateek-giddh — APPROVED** [review]

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 2]
```diff
@@ -9362,32 +9362,79 @@ input[type="checkbox"].cdk-visually-hidden::after {
...
+        display: block !important;
+    }
+}
 @media screen and (max-width: 768px) {
     .custom-mat-tabs {
         .mat-tab-header {
             display: block !important;
         }
     }
+    .form-select {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/style-2.scss:9389]: add parent class

File: apps/web-giddh/src/assets/locale/user-details/en.json [hunk 3]
```diff
@@ -131,7 +131,9 @@
         "search_companies": "Search companies",
         "transaction_used": "transaction used",
         "expiring_within": "Expiring within",
-        "expires": "Expires"
+        "expires": "Expires",
+        "timeline": "Timeline",
+        "search_plans": "Search Plan and Companies"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/locale/user-details/en.json:136]:
```suggestion
        "search_plans": "Search Plans and Companies"
```

File: apps/web-giddh/src/app/userDetails/components/move-company/move-company.component.scss [hunk 4]
```diff
@@ -1,7 +1,23 @@
...
+    margin: 0 !important;
+    padding: 0 !important;
+}
+
+.move-wrapper {
+    .mat-input-field {
+        float: left;
+        width: 100%;
+        height: 37px;
+        border: 1px solid #e2e2e2;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/userDetails/components/move-company/move-company.component.scss:16]: use color variable

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**